src/scalper.ts

```diff
@@ -181,13 +195,31 @@ async function scalperPerp() {
     'sell',
     fairValue * (1 + hrStdDev),
     netGamma,
-    { orderType: 'limit'},
+    { orderType: 'postOnly', expiryTimestamp: getUnixTs() + scalperWindow-1, clientOrderId: gammaAskID},
   );
-  console.log('Bid', gammaBid)
-  console.log('Ask', gammaAsk)
-
-  // TODO Start 1 hour timer
-
+  console.log('Bid', gammaBid, 'ID', gammaBidID)
+  console.log('Ask', gammaAsk, 'ID', gammaAskID)
+
+  // Check by periods per scalperWindow for fills matching either gamma scalp and rerun after scalperWindow expires
+  let periods = 180;
+  let timeWaited = 0;
+  let filledBidGamma: number;
+  let filledAskGamma: number;
+  while (timeWaited < scalperWindow){
+    // Check this was buggy here updating account orders
+    // mangoAccount = (await client.getMangoAccountsForOwner(mangoGroup, owner.publicKey))[0];
```

Comment:
remove unneeded comments like this

Reply:
Will remove later, for now just for benefit of the Mango team

Reply:
// TODO Risk Manager feeds in new DIP position automatically
// Risk Manager sends new DIP positions upon User deposit into DIPs without MM fill
// See DIP object below for fields

Sufficient?